subscription_test.go

```diff
@@ -121,8 +121,8 @@ func TestSchemaSubscribe(t *testing.T) {
 				helloSaidResolver: &helloSaidResolver{
 					upstream: closedUpstream(
 						&helloSaidEventResolver{msg: "Hello world!"},
-						&helloSaidEventResolver{err: resolverErr},
 						&helloSaidEventResolver{msg: "Hello again!"},
+						&helloSaidEventResolver{err: resolverErr},
```

Comment:
Why is this test changed? Shoudln't you leave it intact and add new tests that used to panic before but don't do it anymore after this fix?